<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPanelDialog</class>
 <widget class="QDialog" name="AddPanelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Panel</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>550</y>
     <width>400</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="positionLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>210</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Panel position</string>
   </property>
  </widget>
  <widget class="QComboBox" name="position">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>430</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Top</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bottom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Left</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Right</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="screenLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>480</y>
     <width>210</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Screen</string>
   </property>
  </widget>
  <widget class="QComboBox" name="screen">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>475</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="componentsLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>68</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Optional features</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showApplicationMenu">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Application Menu</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="showPager">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Pager</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="headerLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>361</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Please set up your new panel.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showClock">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Clock</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="showTaskManager">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>311</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Task Manager</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="styleLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>210</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Dock style</string>
   </property>
  </widget>
  <widget class="QComboBox" name="style">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>385</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Glass 3D</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Glass 2D</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Flat 2D</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Metal 2D</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="showTrash">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Trash</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="showVolumeControl">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume Control</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="showVersionChecker">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Version Checker</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddPanelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddPanelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
